lib/adapters/class-vip-enterprise-search.php

```diff
@@ -18,15 +19,109 @@
  */
 class VIP_Enterprise_Search extends Adapter {
 
+	/**
+	 * Add facets to EP query.
+	 * Filters `ep_post_formatted_args`.
+	 *
+	 * @param array $query
+	 * @return array
+	 * @see \ElasticPress\Indexable\Post\Post.
+	 */
+	public function add_facets_to_ep_query( $query ) {
+		// Do we have any facets specified?
+		$searched_facets = get_query_var( 'fs' );
```

Comment:
good point. do you have a suggestion on how we might account for headless projects? cc @kevinfodness

Reply:
Yeah, we could make a generic function that accepts searched facets as an argument, which could be populated here using `get_query_var` or could be populated from a request to GraphQL or the REST API's search endpoint, and could also be filtered. Adding support for a standard structure in REST and GraphQL makes sense, and then the filter would allow for further customization on a per-site basis. However, this feels like something we can add as an enhancement later vs. being a blocker for merging this PR.